tests/gis_tests/test_geoforms.py

```diff
@@ -346,6 +349,10 @@ class PointForm(forms.Form):
             self.assertNotIn(
                 "options['default_lat'] = %d;" % forms.OSMWidget.default_lat,
                 rendered)
+        if forms.OSMWidget.default_zoom != 17:
```

Comment:
Agreed, I'll suggest https://github.com/django/django/pull/8493 and then we can rebase this and use that approach if it's okay.